### PR TITLE
Add fr glossary

### PR DESCRIPTION
**Pull request by Dufgui**

### CODE REVIEW

**Comment by skyluc** [comment]
A bit busy for a couple of days, but I will look at this on Friday.

**Comment by jboner** [comment]
@skyluc Did you have a chance to review this PR?

**Comment by Dufgui** [comment]
any news? @skyluc ?

File: glossary.fr.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+## <a name="Scalabilité"></a>Scalabilité
+The ability of a [système](#Système) to make use of more computing [ressources](#Ressource) in order to increase its performance is measured by the ratio of throughput gain to resource increase. A perfectly scalable system is characterized by both numbers being proportional: a twofold allocation of resources will double the throughput. Scalability is typically limited by the introduction of bottlenecks or synchronization points within the system, leading to constrained scalability, see [Amdahl’s Law and Gunther’s Universal Scalability Model](http://blogs.msdn.com/b/ddperf/archive/2009/04/29/parallel-scalability-isn-t-child-s-play-part-2-amdahl-s-law-vs-gunther-s-law.aspx).
```
Comment (skyluc) [inline glossary.fr.md]: This section was not translated.

**Review by skyluc — CHANGES_REQUESTED** [review]
Let me start by saying that this is good work. I know how hard this kind of translation can be.
I found a few problems and typos.

One bigger problem I think, is the translation of the link anchor ids. I think those anchor ids should stay being in English, so the same anchors are used in all glossary translation.
Also, they are currently only translated in the glossary file, not in the manifesto file.

The items under 'other suggestions:' are just that, suggestions that I don't feel strongly about.

File: glossary.fr.md (hunk 1 above)
Comment (jboner) [reply glossary.fr.md:88]: Ok. Probably an oversight. @Dufgui can you add it? Or any other takers? @skyluc?

**Comment by Dufgui** [comment]
Hello, sorry for the delay, but all reviews/remarks are now take account.

**Comment by jboner** [comment]
Thanks a lot. 
@skyluc Can you review these changes?

**Comment by Dufgui** [comment]
please reaview @skyluc  ;o)

File: README.fr.md [hunk 2]
```diff
@@ -7,14 +7,14 @@ Ces changements sont causés par l’évolution rapide de ce que l’on attend d
...
 
-Des systèmes développés selon ces directives sont plus flexibles, à couplage faible et [extensibles](/glossary#Scalability). Cela les rend plus simples à développer et plus propices au changement. Ils ont une plus grande tolérance vis-à-vis des [erreurs](/glossary#Failure), et en cas d’erreur, réagissent de façon plus élégante, évitant ainsi des pannes catastrophiques. Ayant des temps de réponse rapide, les systèmes réactifs permettent aux [utilisateurs](/glossary#User) une interaction accrue et ainsi une plus grande satisfaction.
+Des systèmes développés selon ces directives sont plus flexibles, à couplage faible et [extensibles](/fr/glossary#Scalabilité). Cela les rend plus simples à développer et plus propices au changement. Ils ont une plus grande tolérance vis-à-vis des [erreurs](/fr/glossary#Échec), et en cas d’erreur, réagissent de façon plus élégante, évitant ainsi des pannes catastrophiques. Ayant des temps de réponse rapide, les systèmes réactifs permettent aux [utilisateurs](/fr/glossary#Utilisateur) une interaction accrue et ainsi une plus grande satisfaction.
```
Comment (adriaanm) [inline README.fr.md:10]: The translated links seem inconsistent with the translated glossary, which leaves the anchor names in English, so perhaps just leave them in English here too? I'm not familiar with how these pages are generated, but I assume the link `[...](/fr/glossary#Scalabilité)` needs to match up with the `name="Scalability"` [below](https://github.com/reactivemanifesto/reactivemanifesto/pull/130/files#diff-61d7c8074220aea1e6b6e026801316b2315f03add939dcce0d6c7b06531b906cR87)

Comment (adriaanm) [inline README.fr.md:10]: Concretely:

```suggestion
Des systèmes développés selon ces directives sont plus flexibles, à couplage faible et [extensibles](/fr/glossary#Scalability). Cela les rend plus simples à développer et plus propices au changement. Ils ont une plus grande tolérance vis-à-vis des [erreurs](/fr/glossary#Failure), et en cas d’erreur, réagissent de façon plus élégante, évitant ainsi des pannes catastrophiques. Ayant des temps de réponse rapide, les systèmes réactifs permettent aux [utilisateurs](/fr/glossary#User) une interaction accrue et ainsi une plus grande satisfaction.
```

**Comment by jboner** [comment]
Thanks, @Dufgui, and @adriaanm. Merging.

**Comment by jboner** [comment]
@rasummer Can you update the site with the new French glossary?

**Comment by rasummer** [comment]
@jboner sorry for delayed response, I've been on vacation. Yes I will add to list and let you know once done.

**Comment by rasummer** [comment]
This has been updated on the website

**Pull request merged**